index.html

```diff
@@ -0,0 +1,928 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>New York Times</title>
+    <link rel="stylesheet" href="./assets/css/index.css" />
+  </head>
+  <body>
+    <div class="fixed-nav">
+      <nav class="nav-bar">
+        <div class="left">
+          <img
+            class="menu-icon"
+            src="assets/img/menu-icon.svg"
+            alt="menu-icon"
+          />
+          <img src="assets/img/search-icon.svg" alt="search-icon" />
+          <span
+            ><a href="https://www.nytimes.com/section/science/space"
+              >SPACE & COSMOS</a
+            ></span
+          >
+        </div>
+        <div class="center">
+          <img src="assets/img/brand.svg" alt="brand-icon" />
```

Comment:
- [ ] Kindly enclose this image inside an anchor tag `<a><a>` as the original page has it.